RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -3773,7 +3778,7 @@ boolean auto_change_mcd(int mcd)
 		best = 11;
 	}
 
-	if(my_level() >= 13 && !get_property("auto_disregardInstantKarma").to_boolean())
+	if(my_level() == 13 && !get_property("auto_disregardInstantKarma").to_boolean())
```

Comment:
why?

Reply:
If we are level 14+ then regardless of the caring about karma there is no reason to intentionally avoid gaining XP as we already missed it.

Reply:
Add comment explaining this

Reply:
comment added